spec/models/group_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  before :all do
+    User.destroy_all
+    @user = User.new(name: 'Kasuki', email: 'first@email.com', password: '123456',
+                     password_confirmation: '123456')
+    @user.skip_confirmation!
```

Comment:
- [ ] All your tests seem to be failing due to this method (for example I could not generate the seed data until I removed this line as well). However, even when I remove this line the tests still fail anyway. I would recommend you fix this

  ![herrera](https://user-images.githubusercontent.com/36542986/170684532-168dd734-f2b6-4cc4-b41d-bd921761ea85.jpg)

Reply:
Oh sorry, I made some changes, and I forgot to fix the tests. 😵‍💫, I just corrected those issues, and now they are working.